framework/src/Volo.Abp.RabbitMQ/Volo/Abp/RabbitMQ/AbpRabbitMqExchangeAndQueueOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Volo.Abp.RabbitMQ;
+
+public abstract class AbpRabbitMqExchangeAndQueueBaseOptions
+{
+    public string? ConnectionName { get; set; }
+
+    public string? ClientName { get; set; }
+
+    public bool QueueDurable { get; set; }
+
+    public bool QueueExclusive { get; set; } = true;
+    
+    public bool QueueAutoDelete { get; set; } = true;
```

Comment:
It changes the previous default value

<img width="514" alt="image" src="https://github.com/abpframework/abp/assets/16813853/9e04c9e6-c1c9-4933-986e-39255791fcae">